operate/client/src/App/ProcessInstance/TopPanel/index.tsx

```diff
@@ -365,8 +368,10 @@ const TopPanel: React.FC = observer(() => {
                     state={payload.flowNodeState}
                     count={payload.count}
                     container={overlay.container}
-                    isFaded={modificationsStore.hasPendingCancelOrMoveModification(
+                    isFaded={hasPendingCancelOrMoveModification(
                       overlay.flowNodeId,
+                      undefined,
+                      modificationsByFlowNode,
```

Comment:
We could consider passing an object as arguments because right now it's a bit confusing what some items mean. Like `undefined`, we don't know what it represents